src/app/tvshows/store/tvshows.selectors.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { select } from '@angular-redux/store';
+import { Observable } from 'rxjs/Observable';
+import { TvShowData } from './tvshows.models';
+import { RootState } from '../../store';
+
+export class TvShowSelectors {
+  @select(getSortedResults) public readonly tvshows$: Observable<TvShowData[]>;
```

Comment:
Just checking if you have a consensus on whether to put decorators inline or on the line above. I tend to put them on the line above (so in this case I'd have 5 lines (2 decorator / function pairs & a line break to separate them). Just make sure your consistent everywhere & consider seeing if you can add a linting rule to enforce this.

Reply:
Will do.  I eventually want to make these arrow functions and in-line with the decorators, but a bug in the current version of typescript is preventing me from doing this.